go/libkb/resolve.go

```diff
@@ -171,6 +171,7 @@ func (r *Resolver) resolveURLViaServerLookup(au AssertionURL, input string, with
 
 	var them *jsonw.Wrapper
 	if them, res.err = ares.Body.AtKey("them").ToArray(); res.err != nil {
+		res.err = NotFoundError{fmt.Sprintf("User not found for query: %q", input)}
```

Comment:
We don't have a SCDeleted constant that maps to status_codes.iced DELETED, but it's never come up before.  Is `thurnish41ey` deleted?